src/toolbar/ToolbarListButton.tsx

```diff
@@ -87,7 +84,7 @@ export function ToolbarListButton<E>({
                 {buttonContent}
             </ToolbarButtonView>
             <Popup anchorElement={anchorElement} open={popupOpen} onOpenChange={hide}>
-                <Menu size="l" className={b('menu')} qa={qaMenu}>
+                <Menu size="l" className={b('menu')} qa="g-md-toolbar-additional">
```

Comment:
`qa` attribute for this Menu should be set from props, since it is a generic component and is used for other lists as well.